src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public abstract class Task {
+    private String name;
+    private boolean done;
```

Comment:
Name and done should be protected instead, so that classes that extends it can use it